.gitignore

```diff
@@ -20,7 +20,7 @@ test/
 
 media/
 
-/test.py
+*/test.py
```

Comment:
If it's not recursive yet, put double stars here

Reply:
հը՞ ։դ

Reply:
եթե ճիշտ եմ հասկանում, ռեկուրսիվ ա, մի քանի ֆոլդրի խորությամբ test.py որ սարքում եմ, իգնորում ա

Reply:
հմ, ճիշտ ես, ես սխալվեցի ։դ